src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -21,148 +84,1642 @@ const ActorInit En_Ossan_InitVars = {
+                                this->stickRightPrompt.stickColorB, this->stickRightPrompt.stickColorA,
+                                this->stickRightPrompt.stickTexX, this->stickRightPrompt.stickTexY,
+                                this->stickRightPrompt.texZ, 0, 0, 1.0f, 1.0f);
+        }
+    }
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+
+s32 EnOssan_OverrideLimbDrawCuriosityShopMan(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos,
+                                             Vec3s* rot, Actor* thisx) {
+    EnOssan* this = THIS;
+
+    if (limbIndex == 16) {
+        SysMatrix_InsertXRotation_s(this->headRot.y, MTXMODE_APPLY);
+    }
+    return false;
+}
+
+s32 EnOssan_OverrideLimbDrawPartTimeWorker(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                                           Actor* thisx) {
+    EnOssan* this = THIS;
+
+    if (limbIndex == 15) {
+        SysMatrix_InsertXRotation_s(this->headRotPartTimeWorker.y, MTXMODE_APPLY);
+        SysMatrix_InsertZRotation_s(this->headRotPartTimeWorker.x, MTXMODE_APPLY);
+    }
+    return false;
+}
+
+void EnOssan_PostLimbDrawCuriosityShopMan(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot,
+                                          Actor* thisx) {
+    EnOssan* this = THIS;
+
+    if (limbIndex == 1 || limbIndex == 9 || limbIndex == 12) {
+        rot->y += (s16)Math_SinS(this->limbRotTableY[limbIndex]) * 200;
+        rot->z += (s16)Math_CosS(this->limbRotTableZ[limbIndex]) * 200;
+    }
+}
+
+void EnOssan_PostLimbDrawPartTimeWorker(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot,
+                                        Actor* thisx) {
+    static Vec3f sPartTimeWorkerFocusOffset = { 800.0f, 500.0f, 0.0f };
+    EnOssan* this = THIS;
+
+    if (limbIndex == 15) {
+        SysMatrix_MultiplyVector3fByState(&sPartTimeWorkerFocusOffset, &this->actor.focus.pos);
+    }
+}
+
+void EnOssan_DrawCuriosityShopMan(Actor* thisx, GlobalContext* globalCtx) {
+    static TexturePtr* sEyeTextures[] = { &D_06005BC0, &D_06006D40, &D_06007140 };
+    EnOssan* this = THIS;
+    s32 pad;
+
+    OPEN_DISPS(globalCtx->state.gfxCtx);
+    func_8012C28C(globalCtx->state.gfxCtx);
+    gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(sEyeTextures[this->eyeTexIndex]));
+    SkelAnime_DrawSV(globalCtx, this->skelAnime.skeleton, this->skelAnime.limbDrawTbl, this->skelAnime.dListCount,
+                     EnOssan_OverrideLimbDrawCuriosityShopMan, EnOssan_PostLimbDrawCuriosityShopMan, &this->actor);
+    EnOssan_DrawCursor(globalCtx, this, this->cursorX, this->cursorY, this->cursorZ, this->drawCursor);
+    EnOssan_DrawStickDirectionPrompts(globalCtx, this);
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+
+void EnOssan_DrawPartTimeWorker(Actor* thisx, GlobalContext* globalCtx) {
+    static TexturePtr* sEyeTextures[] = { &D_06006498, &D_06006B18, &D_06006F18 };
```

Comment:
Since `TexturePtr` is already a pointer, making an array of pointers to it doesn't make much sense, right?
```suggestion
    static TexturePtr sEyeTextures[] = { D_06006498, D_06006B18, D_06006F18 };
```